documentation/asciidoc/accessories/iqaudio/configuration.adoc

```diff
@@ -52,34 +56,33 @@ file will need to be written to the Pi’s SD card to allow the Pi to boot.
 
 === Attaching the HAT
 
-The IQaudio range of sound cards attach to the Raspberry Pi’s 2x20 way pin header (GPIO header).
-They are designed to be supported on the Pi using the supplied PCB standoffs and screws. In
-general no soldering is required to the IQaudio boards for normal operation.
+The Raspberry Pi range of sound cards attach to the Raspberry Pi’s 2x20 way pin header (GPIO header).
```

Comment:
i think I normally write 40-pin header, mainly because there aren't 40 GPIO pins, so writing 40-pin GPIO header is technically incorrect. Which is the best type of incorrect.

Reply:
We definitely don't refer to it anywhere as 2x20 pins.